masking-examples/consistent-format-preserve/python-udf.py

```diff
@@ -0,0 +1,55 @@
+import hashlib
+import random
+
+
+# consistent_mask is a reference implementation of a masking function that
+# preserves the format of the input data consistently, such that the same
+# input will always produce the same output. It replaces each alphanumeric
+# character with a random character of the same type (uppercase, lowercase,
+# or digit), while leaving all other characters unchanged.
+def consistent_mask(data):
```

Comment:
I would suggest specifying type hints to document what argument types we support. Also, we could support both integer and string types (convert the integer to string (base 10), mask, and then convert back to integer)